common/decklist.cpp

```diff
@@ -636,32 +637,40 @@ struct WriteToStream {
        if (prefixSideboardCards && node->getName() == DECK_ZONE_SIDE) {
            stream << "SB: ";
        }
-       stream << QString("%1 %2\n").arg(
-           card->getNumber()
-       ).arg(
-           card->getName()
-       );
+       if (!slashTappedOutSplitCards) {
+           stream << QString("%1 %2\n").arg(
+               card->getNumber()
+           ).arg(
+               card->getName()
+	   );
+       } else {
+	   stream << QString("%1 %2\n").arg(
+	       card->getNumber()
+	   ).arg(
+               card->getName().replace("//", "/").replace("Who / What / When / Where / Why", "Who/What/When/Where/Why")
```

Comment:
Instead of hardcoding a specific card name, what about doing `.replace("//", "/").replace(" / ", "/")`? Would that work?

Reply:
I don't think so, because we have it as `Who // What // When // Where // Why` with double `//` in our database, the same as all other split cards.

Check my other post for more details @Daenyth!